README.rst

```diff
@@ -179,7 +179,7 @@ You can find a quick demo of coala here:
 
 |asciicast|
 
-.. |asciicast| image:: https://asciinema.org/a/42968.png
+.. |asciicast| image:: http://i.imgur.com/rLasBoQ.png
    :target: https://asciinema.org/a/42968?autoplay=1
    :width: 100%
```

Comment:
Please use the width property, so that if in the future we want to update the width again, we don't have to upload another image.
Thanks.